samples/go/csv/read_test.go

```diff
@@ -148,6 +148,9 @@ func TestEscapeStructFieldFromCSV(t *testing.T) {
 		"https://picasaweb.google.com/data", "httpspicasawebgooglecomdata",
 		"💩", "",
 		"11 1💩", "",
+		"-- A B", "aB",
+		"-- A --", "A",
```

Comment:
uncertain if these expectations are the right one , I can change it if it makes more sense going in a different direction